nmdc_schema/nmdc_materialized_patterns.schema.json

```diff
@@ -7449,11 +7609,8 @@
                 },
                 "part_of": {
                     "description": "Links a resource to another resource that either logically or physically includes it.",
-                    "items": {
-                        "pattern": "^nmdc:sty-[0-9][a-z]{0,6}[0-9]-[A-Za-z0-9]{1,}(\\.[A-Za-z0-9]{1,})*(_[A-Za-z0-9_\\.-]+)?$",
-                        "type": "string"
-                    },
-                    "type": "array"
+                    "pattern": "^nmdc:datgen-[0-9][a-z]{0,6}[0-9]-[A-Za-z0-9]{1,}(\\.[A-Za-z0-9]{1,})*(_[A-Za-z0-9_\\.-]+)?$",
```

Comment:
We decided not to have a typecode for datgen, instead there are typecodes dgnt and dtms for the subclasses of data generation